sdk/examples/client/block/01_block_confirmation_time.rs

```diff
@@ -34,8 +34,6 @@ async fn main() -> Result<()> {
 
     println!("{block:#?}");
 
-    // Try to check if the block has been confirmed.
-    client.retry_until_included(&block_id, None, None).await?;
```

Comment:
No replacement?

Reply:
For the client I wouldn't add a high level reissue method, since this would require a secret manager and also enough  allotted mana before and without checking this it can easily go negative

If you're just speaking about this place in the example I could add a sleep of 5 seconds or a loop which checks the metadata of the block to see if it got confirmed? But the BlockMetadataResponse is not updated yet

Reply:
Yeah it feels like this might break the example

Reply:
Added a loop which checks for inclusion state